lib/tokenizers.js

```diff
@@ -60,30 +61,42 @@ export async function get({id, explain = false} = {}) {
 
   const record = await collection.findOne(query, {projection});
   if(!record) {
-    const details = {
-      httpStatusCode: 404,
-      public: true
-    };
-    throw new BedrockError(
+    const error = new BedrockError(
       'Tokenizer not found.',
-      'NotFoundError', details);
+      'NotFoundError', {httpStatusCode: 404, public: true});
+    logger.error(error);
```

Comment:
Can we get confirmation from @davidlehn and @mattcollier that just passing `error` like this without an error message is going to be ok? I'm not sure we do this in other places and it may not surface the information we want/need in the same manner.

Reply:
do we have test coverage for this?  If so, running the test suite with something like
```
npm test test --log-level debug
```
should reveal how the error is being handled as logger.error should appear on the console when the corresponding test runs.

Other than that, yes, this is a non-standard pattern of use for `logger.error`.

Reply:
I'm pretty sure that won't do what you want.  Will the error just get thrown up the stack and get logged anyway?  If you want to log it here, probably do something like:
```suggestion
    logger.error(error.message, {error});
```
I see there are a few other places with the same issue.